appendix/resources/readme_badges.md

```diff
@@ -0,0 +1,112 @@
+---
+title: Readme Badges
```

Comment:
maybe we could generalize such a page to "Promotion" and include information about embeddable download widget, links to relevant parts of marketplace docs (custom pages, measuring downloads, ...)?